src/cmd/build.js

```diff
@@ -4,14 +4,14 @@ import {createWriteStream} from 'fs';
 
 import {fs} from 'mz';
 import parseJSON from 'parse-json';
-import eventToPromise from 'event-to-promise';
+import {eventToPromise as defaultEventToPromise} from 'event-to-promise';
```

Comment:
@harikishen you forgot to apply this change in the last update on this patch.

`eventToPromise` is not a named export of the 'event-to-promise' npm package, it is its default export and it should be imported with `defaultEventToPromise` as its name using the syntax:
```js
import defaultEventToPromise from 'event-to-promise';
```